packages/datadog-plugin-openai/src/index.js

```diff
@@ -15,6 +15,14 @@ const RE_TAB = /\t/g
 // TODO: In the future we should refactor config.js to make it requirable
 let MAX_TEXT_LEN = 128
 
+let encodingForModel
+try {
```

Comment:
APM folks - let me know if you have thoughts on this implementation. explained more in the PR description, but TL;DR in the Python tracer, we do something similar where we [`try/catch` importing](https://github.com/DataDog/dd-trace-py/blob/b93575e72726566c6eb90cf81e012aba846b7439/ddtrace/contrib/openai/utils.py#L13-L18) this [`tiktoken`](https://www.npmjs.com/package/tiktoken) library. I included it here for consistency, and from customer ask. I'd prefer to have it as an option, but didn't want to explicitly make it an optional dependency for `dd-trace-js`, so if there are any objections/other ways of doing this, happy to implement!

Reply:
This is totally fine.